keylime/da/attest.py

```diff
@@ -148,11 +153,20 @@ def main() -> None:
                         f_e_id,
                         f_e_ctx,
                     )
-                    return
+                    break
             else:
                 logger.info(
                     '---------- Agent %s was in "attested" state at %s (TPM delta: %s)',
                     agent_uuid,
                     attestation_record["verifier" + "_timestamp"],
                     d_tpm_ts,
                 )
+
+    if failed_agents:
+        logger.info(
+            "=> The following agents failed attestation at some point in recorded history: %s", ",".join(failed_agents)
+        )
+        sys.exit(255)
```

Comment:
Usually an error is signed as `sys.exit(1)` in this codebase (there are some -1, that correspond to 255, and one 2).  Unless we want to provide meaning to the exit code (in that case I suggest to use os.EX_* family), we should normalize it to 1 here too.

Reply:
Make sense.